tox.ini

```diff
@@ -22,6 +22,7 @@ envlist = fix_lint,
           coverage
 skip_missing_interpreters = true
 requires =
+    setuptools>=40.6.0
```

Comment:
This shouldn't be required. It's only required at build time, for which we use PEP-517 and this is defined here: https://github.com/snowflakedb/snowflake-connector-python/blob/93c9b69f9e2d7d8b159c24c85ab775e54bc1be0c/pyproject.toml#L5

Reply:
Yes, but I think the problem is that tox tries to build the package first, and it uses its `.tox/.tox` provisioned environment to do this.  However, that environment doesn't have setuptools installed by default, so if you do a clean git clone of the repository and then try e.g. `tox -e py38`, it fails very early.  That's why this is added to the `[tox]` environment.  If that doesn't happen for you then it's worth looking at why it differs.  It could be that the Python you're using for the tox environment outside of the provisioned one already has setuptools installed.  Or could you be using tox v3 where maybe things work differently?  I'm using tox 4.11.4 from homebrew.

Reply:
I solved this by removing the dependency on `tox-external-wheels`, which I believe pins tox's provisioned version to tox 3 and requires setuptools because only tox 4 knows about pyproject.toml.